### PR TITLE
Update foreman_bootdisk to 10.0.1 [deb]

### PR DESCRIPTION
**Pull request by timogoebel**
Please build for

* [x] Nightly
* [x] 1.16
* [x] 1.15
* [ ] 1.14

### CODE REVIEW

**Comment by mmoll** [comment]
[test]

File: plugins/ruby-foreman-bootdisk/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-bootdisk (10.0.0-1) stable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-foreman-bootdisk/debian/changelog]: this needs a version bump, too

**Comment by timogoebel** [comment]
@mmoll: Thanks, done.

**Comment by mmoll** [comment]
just to get extra sure: is this really compatible with 1.15 core?

**Comment by timogoebel** [comment]
@mmoll: Yes. @lzap kindly tested this.

**Comment by lzap** [comment]
Yes, not this artifact but the code changes @timogoebel made. I can confirm that this works:

https://github.com/theforeman/foreman_bootdisk/commit/0bb5356ea66d52811e3843f39f6a0585539e6c99

**Comment by mmoll** [comment]
perfect, danke @timogoebel!

**Pull request merged**